todolist/todo_test.go

```diff
@@ -21,3 +25,36 @@ func TestValidity(t *testing.T) {
 		t.Error("Invalid todo is being reported as valid")
 	}
 }
+
+//SetUpTestMemoryTodoList sets up a fixtures test todolist
+func SetUpTestMemoryTodoList() *TodoList {
+	store := &MemoryStore{}
```

Comment:
These todos are the todos in the test file, I have moved them in here so its safe from meddling.